tests/integration/cgroups.bats

```diff
@@ -70,5 +70,11 @@ EOF
 
     # update kernel memory limit
     run "$RUNC" update test_cgroups_kmem --kernel-memory 50331648
-    [ ! "$status" -eq 0 ]
+    # Since kernel 4.6, we can update kernel memory without initialization
+    # because it's accounted by default.
+    if [ "$KERNEL_MAJOR" -lt 4 ] || [ "$KERNEL_MAJOR" -eq 4 -a "$KERNEL_MINOR" -le 5 ]; then
+        [ ! "$status" -eq 0 ]
+    else
+        [ "$status" -eq 0 ]
```

Comment:
Isn't `4.3.3` failed as expected?

In my research, the behavior change because of b313aeee2509 ("mm: memcontrol: enable kmem accounting for all cgroups in the legacy hierarchy"), it is in `v4.5-2559-gb313aee`, and I tested on 4.5 and 4.4.3, they all failed as expected. So kernel version older than 4.5 (including) would fail.

(here `fail` I mean update kernel memory without initialization, or do you mean the test case failed on `4.3.3` in your box?)